### ISSUE TITLE
USWDS v3 - Button group / VA DS Button pair - Accessibility review

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1521]
## Description
We need to conduct a full accessibility review of the USWDS v3 Button group component and compare with VA DS Button pair component. 

## Details
[USWDS Button Group component](https://designsystem.digital.gov/components/button-group/)
[VA DS Button pair component](https://design.va.gov/components/button/button-pair)

## Tasks
- [ ] Full accessibility review of the component is completed
- [ ] Add descriptions of any issues found to this ticket
    - [ ] If any problems are found, add screen recordings and/or screenshots to comments on this ticket that show the problem.
- [ ] If necessary, discuss any accessibility issues found with the engineer who developed the component
- [ ] Re-review any fixes the developer makes to the component

## Acceptance Criteria
- [ ] Component has been reviewed for accessibility issues
- [ ] Any accessibility issues have been addressed

**Comment by humancompanion-usds** [comment]
@caw310 - Do we still need this? It's been through CollabCycle at this point.

**Comment by caw310** [comment]
No. Not needed bc both have been through staging review.  I will close.
